test/noFilters.test.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as chai from "chai";
+import * as chaiAsPromised from "chai-as-promised";
+chai.use(chaiAsPromised);
+const expect = chai.expect;
+
+import { FeatureManager, ConfigurationObjectFeatureFlagProvider } from "./exportedApi";
+
+const featureFlagsDataObject = {
+    "feature_management": {
+        "feature_flags": [
+            {
+                "id": "BooleanTrue",
+                "description": "A feature flag with no Filters, that returns true.",
+                "enabled": true,
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "BooleanFalse",
+                "description": "A feature flag with no Filters, that returns false.",
+                "enabled": false,
+                "conditions": {
+                    "client_filters": []
+                }
+            },
+            {
+                "id": "InvalidEnabled",
+                "description": "A feature flag with an invalid 'enabled' value, that returns false.",
+                "enabled": "invalid",

```

Comment:
Is this valid input? I'd expect something like this to throw an error instead of assuming that enabled is false.

Reply:
Agree. It makes more sense to throw an exception. 

I copied this sample from https://github.com/Azure/AppConfiguration/blob/d0ef7d5507b9cee15893eead5861e3b1a82d3efd/docs/FeatureManagement/Samples/NoFilters.sample.json#L38

We should change the description of this sample there.

Reply:
@mrm9084, could you help update the test?
In .NET provider, we throw an exception when "enabled" field is anything other than true/false: 
https://github.com/Azure/AppConfiguration-DotnetProvider/blob/a9eacbc1ba62e12eb1a42c2a2ebb326289f5e0f8/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs#L142C24-L162C30

Reply:
updated.